### PR TITLE
KHP3-1522 - Completed appointments tab

### PR DESCRIPTION
**Pull request by CynthiaKamau**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
- Added base table component to serve tab tables with the same structure
- Added completed appointments table 

## Screenshots
completed

*None.*

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { useAppointmentsMetrics } from './appointment-metrics.resource';
 import MetricsCard from './metrics-card.component';
 import MetricsHeader from './metrics-header.component';
 import styles from './appointments-metrics.scss';
-import { ErrorState } from '@openmrs/esm-framework';
+// import { ErrorState } from '@openmrs/esm-framework';
```
Comment (makombe) [inline packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx:8]: Good work.
I suggest that you drop the commented out line

File: packages/esm-appointments-app/src/appoinments-tabs/completed-appointments.component.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const CompletedAppointments: React.FC = () => {
+  const { appointments, isLoading } = useAppointments();
+  const { t } = useTranslation();
+
+  return (
+    <div>
+      <AppointmentsBaseTable
+        appointments={appointments}
+        isLoading={isLoading}
+        tableHeading={t('completedAppointments', 'Completed Appointments')}
```
Comment (makombe) [inline packages/esm-appointments-app/src/appoinments-tabs/completed-appointments.component.tsx:15]:
```suggestion
        tableHeading={t('completedAppointments', 'Completed appointments')}
```

File: packages/esm-appointments-app/src/appoinments-tabs/booked-appointments.component.tsx [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+const BookedAppointments: React.FC = () => {
+  const { appointments, isLoading } = useAppointments();
+  const { t } = useTranslation();
+
+  return (
+    <div>
+      <AppointmentsBaseTable
+        appointments={appointments}
+        isLoading={isLoading}
+        tableHeading={t('bookedAppointments', 'Booked Appointments')}
```
Comment (makombe) [inline packages/esm-appointments-app/src/appoinments-tabs/booked-appointments.component.tsx:15]:
```suggestion
        tableHeading={t('bookedAppointments', 'Booked appointments')}
```

File: packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx (hunk 1 above)
Comment (makombe) [inline packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx:8]: Remove the commented out import

File: packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx [hunk 4]
```diff
@@ -15,9 +15,9 @@ const AppointmentsMetrics: React.FC = () => {
     return <DataTableSkeleton role="progressbar" />;
   }
 
-  if (isError) {
-    <ErrorState headerTitle={t('errorAppoinmentMetric')} error={isError} />;
-  }
+  // if (isError) {
+  //   <ErrorState headerTitle={t('errorAppoinmentMetric')} error={isError} />;
+  // }
```
Comment (makombe) [inline packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx:20]: Remove the comment out lines of code.

**Review by makombe — APPROVED** [review]

File: packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx (hunk 4 above)
Comment (makombe) [reply packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx:20]: Good work @CynthiaKamau . Please act on those few suggestions thanks.

File: packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx (hunk 1 above)
Comment (CynthiaKamau) [reply packages/esm-appointments-app/src/appointments-metrics/appointments-metrics.component.tsx]: Done

**Pull request merged**